source/use-cases/sensitive-data-encryption.txt

```diff
@@ -75,20 +75,61 @@ velit congue vel.
+By following this section, you learn the difference in output when using a secure client with FLE enabled, and a client without FLE enabled.
+A patient is inserted into the patient’s collection using the following code. 
 
-Ut mattis eros elit, sed ultrices purus lacinia sed. Curabitur tincidunt
-tellus eget neque porta dictum ac vel orci. Vestibulum ante ipsum primis
-in faucibus orci luctus et ultrices posuere cubilia Curae.
 
-Praesent feugiat arcu non risus elementum eleifend. Nullam sed iaculis
-erat.
+.. tabs::
+
+   tabs:
+
+     - id: java-jsonschema-generation
+       name: "Java"
+       content: |
+       
+        .. code-block:: java
+
+            public static void insertPatient(MongoCollection collection, String name, int ssn, String bloodType, ArrayList<Document> medicalRecords, int policyNumber, String provider) {
+                Document patient = new Document();
+                patient.put("name", name);
+                patient.put("ssn", ssn);
+                patient.put("bloodType", bloodType);
+                patient.put("medicalRecords", medicalRecords);
+                Document insurance = new Document();
+                insurance.put("policyNumber", policyNumber);
+                insurance.put("provider", provider);
+                patient.put("insurance", insurance);
+                collection.insertOne(patient);
+            }
+
+A doctor starts entering her new patient’s information in the Medco Management System. When she hits submit the above code is executed using the FLE enabled client. The new patient is inserted into the database, with the sensitive fields encrypted and looks like this:
+
+.. code::
+
+    Document{{_id=5d696fff112d8c439bae7068, name=Peter Parker, ssn=org.bson.types.Binary@afbb5147, bloodType=org.bson.types.Binary@713daeda, medicalRecords=org.bson.types.Binary@113f42c8, insurance=Document{{policyNumber=org.bson.types.Binary@806ab88d, provider=MaestCare}}}}
+
+A receptionist submits another patient’s information in the Medco Management System using the insertPatient method, and the results look like this:
+
+.. code::
+
+    Document{{_id=5d696fff112d8c439bae7068, name=Gwen Stacey, ssn=1112223333, bloodType=a-, medicalRecords=[{weight: 120, bloodPressure: 120/80}], insurance=Document{{policyNumber=4410, provider=PlatinumCare}}}}
+
+.. note::
+    The information that the receptionist inserted is unencrypted even for sensitive data, such as ssn You may want to logic in your application code to prevent from writing to a sensitive field with a client that does not have FLE enabled. Another alternative approach is to use server side field level encryption in addition to CSLFE, this would throw an error if plaintext is attempted to be inserted into a sensitive field.
```

Comment:
- "you may want to ____ logic"
- "to prevent clients not configured for CSFLE from writing to a sensitive field"
- Server-side FLE should probably be the main suggestion rather than an alternative IMO.

Reply:
Resolving since this is well outdated, and we've talked about this in person